public/javascripts/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(<a href="#">Capital A</a>, document.getElementById('buttons'));
+
+ReactDOM.render(<h1>A</h1>, document.getElementById('letters'));
```

Comment:
I'm not super familiar with React, but is it convention or okay to render twice in the same file? Maybe this is a place where you could wrap #buttons and #letters in a parent div and render once?